tests/test_library_client.py

```
@@ -0,0 +1,349 @@
+        """Test that index is cached after successful fetch."""
+        with tempfile.TemporaryDirectory() as tmpdir:
+            client = LibraryClient(cache_dir=Path(tmpdir))
+
+            mock_response = MagicMock()
+            mock_response.read.return_value = json.dumps(SAMPLE_INDEX).encode()
+            mock_response.__enter__ = lambda s: s
+            mock_response.__exit__ = MagicMock(return_value=False)
+
+            with patch("urllib.request.urlopen", return_value=mock_response) as mock_urlopen:
+                # First call
+                index1, from_cache1 = client.fetch_index()
```

Ruff (0.14.14)

[warning] Unpacked variable `index1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)